### ISSUE TITLE
Automatically open default system browser

### ISSUE THREAD

**Issue by kda-jt** [okta/okta-aws-cli#19]
Hi folks 👋🏼 

## Current behaviour

Currently, when `okta-aws-cli` is run, it returns a URL & asks the user to copy it in their browser.

```
$ okta-aws-cli --org-domain ORG_DOMAIN --aws-iam-idp IDP_ARN --aws-acct-fed-app-id APP_ID --oidc-client-id CLIENT_ID

Open the following URL to begin Okta device authorization for the AWS CLI.

https://ORG_DOMAIN/activate?user_code=USERCODE
```
And since the command is interactive, I cannot pipe the URL to `xdg-open` or `open`.

## Expected behaviour

It would be great if it directly opened the browser on the right page.

---
Thanks!

**Comment by nickgamb** [comment]
We are adding this as a configurable option.

**Comment by monde** [comment]
Implemented in #21
